setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='cs251tk',
-    version='2.2.0',
+    version='2.2.1',
```

Comment:
This should probably go to v2.3.0, since it's a new feature and not just a bugfix.